### ISSUE TITLE
Publish-BcContainerApp is hanging up if published to SaaS

### ISSUE THREAD

**Issue by DavidFeldhoff** [microsoft/navcontainerhelper#2661]
**Describe the issue**
Hi Freddy, if I publish a larger app to a BC Online Sandbox, then the Publish-BcContainerApp Command is hanging up. In AppInsights I can see that the installation was done after two minutes and I could verify it on the Sandbox itself as well, but the pipeline was still waiting for the installation to complete. The Sandbox was completely blank before, so each installation was a real installation. No update.

**Scripts used to create container and cause the issue**
```
$authContext = New-BcAuthContext -tenantID $TenantID -refreshToken $refreshtoken
foreach($appFile in $appFiles) {
  Publish-BcContainerApp -appFile $appFile -tenant $TenantId -environment $Environment -bcAuthContext $authContext
}
```

**Full output of scripts**
![image](https://user-images.githubusercontent.com/53570297/187683872-42d2b54a-0533-4a52-aee2-1e0ae6f09440.png)
![image](https://user-images.githubusercontent.com/53570297/187685007-832f92eb-f2a3-49d4-8000-c4445472d7ec.png)

**Additional context**
- does it happen all the time? I can have an eye on it, it was the first time that I tried it.
- did it use to work?

Best regards
David

**Comment by freddydk** [comment]
Might be a timeout - will check

**Comment by DavidFeldhoff** [comment]
Thanks if you need something from me, just feel free to reach out. I think I could send you the app file e.g., if that's helpful

**Comment by freddydk** [comment]
If possible, I would like the collection of Apps you are using to make this fail.
Then I can create a repro and find out why this would fail

Thanks

**Comment by DavidFeldhoff** [comment]
You should've received a mail

**Comment by freddydk** [comment]
Shipped in 4.0.3

**Comment by dsaveyn** [comment]
Hi Freddy,

We have had the same issue for a while now. I noticed we were on version 4.0.2, so I updated BCContainerHelper on our server to version 4.0.5.

Unfortunately the issue still occurs:

![image](https://user-images.githubusercontent.com/92365904/193304427-8f0dbc01-0878-488d-bf00-cd96520d8c75.png)

I checked the apps in BC and the app in question is already installed and so are the apps that depend on it:
![image](https://user-images.githubusercontent.com/92365904/193304633-87a23c99-330b-4f48-9271-ec1eda173057.png)

The app is a small app but most of our apps depend on it. The issue also doesn't always occur, I would say 10% of the time it works.
